cmd/fleetctl/mdm.go

```diff
@@ -90,7 +90,7 @@ func mdmRunCommand() *cli.Command {
 			var (
 				hostUUIDs     []string
 				notFoundCount int
-				platform      string
+				mdmPlatform   string // "darwin" or "windows"
```

Comment:
Nit but as discussed originally, I think this would benefit from having a distinct type (e.g. `fleet.MDMPlatformName` or something like that) with constants defined for darwin and windows so when reading code we can easily see what are the possible values.

Reply:
(and then use those platform constants throughout instead of "darwin" or "windows")

Reply:
Will add at a later iteration to have this merged sooner. Let me know if that's a-ok.

Reply:
Sounds fine to me, thanks!